scripts/prepare-fulfillment-service.sh

```
@@ -37,26 +37,39 @@ if [[ -n "${INSTALLER_VM_TEMPLATE}" || -n "${INSTALLER_CLUSTER_TEMPLATE}" ]]; th
     for attempt in $(seq 1 30); do
         JT_ID=$(curl -kfsS -H "Authorization: Bearer ${AAP_TOKEN}" \
             "${AAP_URL}/api/controller/v2/job_templates/?name=osac-publish-templates" 2>/dev/null | jq -er '.results[0].id // empty' 2>/dev/null) && break
         echo "  attempt ${attempt}/30 - AAP controller API not ready, retrying in 10s..."
         sleep 10
     done
     [[ -z "${JT_ID:-}" ]] && { echo "Failed to find osac-publish-templates AAP job template after 30 attempts"; exit 1; }
+    PROJECT_ID=$(curl -kfsS -H "Authorization: Bearer ${AAP_TOKEN}" \
+        "${AAP_URL}/api/controller/v2/job_templates/${JT_ID}/" 2>/dev/null | jq -r '.project // empty') || PROJECT_ID=""
+    if [[ -n "${PROJECT_ID}" ]]; then
+        echo "  Waiting for AAP project ${PROJECT_ID} to have a valid revision..."
+        retry_until 300 10 '[[ "$(curl -kfsS -H "Authorization: Bearer '"${AAP_TOKEN}"'" \
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)